assets/src/js/header.js

```diff
@@ -108,6 +108,29 @@ const closeElement = (event, buttonClass) => {
   closeButton.click();
 };
 
+const setupCountrySelectorBanner = () => {
```

Comment:
Can we also add this JS to the child theme?

Reply:
Done!